src/pk.c

```diff
@@ -3857,15 +3858,15 @@ static int wolfssl_rsa_sig_encode(int hashAlg, const unsigned char* hash,
         ret = 0;
     }
 
-    if ((ret == 1) && (hashAlg != NID_undef) &&
-            (padding == RSA_PKCS1_PADDING)) {
+    if ((ret == 1) && (hashAlg != wc_NID_undef) &&
```

Comment:
wc_NID_undef => WC_NID_undef

Reply:
`wc_` makes more sense than `WC_` -- they're not macros, they're enums, and they have lower case morphology.

Reply:
done -- turns out they were all macros.  I misremembered them as `enum`s.